raiden/tests/unit/test_startup.py

```diff
@@ -213,7 +216,7 @@ def test_setup_proxies_all_addresses_are_given(routing_mode):
         'services': {},
     }
     contracts = {}
-    blockchain_service = MockChain(network_id=network_id)
+    blockchain_service = MockChain(network_id=network_id, node_address=make_address)
```

Comment:
This should be calls to `make_address` right? Like `node_address=make_address()`

Reply:
hm yes indeed! It's not used at all in those tests, so it did not hit anywhere but I will fix it asap.